signal_analog/flow.py

```diff
@@ -599,6 +666,67 @@ def rateofchange(self):
         return self
 
 
+class RollupType(Enum):
+    """The Roll-up Type for SignalFlow
+
+    See [SignalFx Documentation on Roll-ups](https://docs.signalfx.com/en/latest/charts/resolution-rollups.html)
+    """
+
+    def __str__(self):
+        """
+        Convert to string representation expected by SignalFx
+        """
+        return '"' + self.value + '"'
```

Comment:
Not sure this was the best way to implement but it kept things backwards compatible since all existing code is using a string for the rollup.

Reply:
Makes sense to me!